### ISSUE TITLE
the objects of the tools category in the chests

### ISSUE THREAD

**Issue by G6bor** [tomkeuper/BedWars1058#345]
### Make sure to check before creating a Bug Report

- [X] The issue isn't already found on the Issue tracker.
- [X] My issue is about **BedWars2023** and not any addon.
- [X] The issue isn't already fixed in a Release or Development Build.
- [ ] I have checked all boxes without reading them.

### Type of problem

Gameplay Bug

### What happened?

axes, picks and shears bought in the shop can be put in the chets[
![2024-12-30_16 29 47](https://github.com/user-attachments/assets/1ba36249-7089-40a8-897f-c5257dd78700)
](url)

and there is also a problem with the wooden sword, but I have already reported it in another issue

### Expected Behaviour

none of them can be put in the chests

### How to Reproduce

simply put it in the chest using the left or right click, it doesn't always work the first time

### Additional Info

- minecraft 1.8 version
- without addon
- tested on these: carbonspigot and pandaspigot

if it's fixed in dev build sorry, I don't have that release in my library, I haven't seen it on discord server either

**Comment by tomkeuper** [comment]
I am quite sure this has been fixed a while back! Please re-open if it happens with the latest dev build
